src/buildblock/ProjData.cxx

```diff
@@ -289,6 +289,22 @@ ProjData::get_related_viewgrams(const ViewSegmentNumbers& view_segmnet_num,
   return RelatedViewgrams<float>(viewgrams, symmetries_used);
 }
 
+//std::vector<float>
```

Comment:
get rid of these comments. file should be unchanged

Reply:
👍